packages/xo-web/src/xo-app/proxies/update-appliance-settings.js

```diff
@@ -0,0 +1,74 @@
+import _, { messages } from 'intl'
+import decorate from 'apply-decorators'
+import Icon from 'icon'
+import React from 'react'
+import SingleLineRow from 'single-line-row'
+import { Col, Container } from 'grid'
+import { form } from 'modal'
+import { generateId } from 'reaclette-utils'
+import { injectIntl } from 'react-intl'
+import { provideState, injectState } from 'reaclette'
+import { updateProxyApplianceSettings } from 'xo'
+
+const UpdateApplianceSettingsModal = decorate([
+  provideState({
+    effects: {
+      onHttpProxyChange(_, { target: { value } }) {
+        this.props.onChange({
+          ...this.props.value,
+          httpProxy: value,
+        })
+      },
+    },
+    computed: {
+      idHttpProxyInput: generateId,
+    },
+  }),
+  injectIntl,
+  injectState,
+  ({ intl: { formatMessage }, effects, state, value }) => (
+    <Container>
+      <SingleLineRow>
+        <Col mediumSize={4}>
+          <label htmlFor={state.idHttpProxyInput} style={{ cursor: 'pointer' }}>
```

Comment:
I'm not sure it's a good idea to change the default cursor behaviour here.

Reply:
Why?